src/Raven.Client/ServerWide/Operations/Certificates/CertificateDefinition.cs

```diff
@@ -12,7 +12,7 @@ public class CertificateDefinition
         public SecurityClearance SecurityClearance;
         public string Thumbprint;
         public string PublicKeyPinningHash;
-        public DateTime NotAfter;
+        public DateTime? NotAfter;
```

Comment:
@ml054 I added this for the certificate view in the Studio, if NotAfter is null the default expiration is still 5 years so nothing is broken and you can add the UI later.

Reply:
@iftahbe when we are putting that into storage (in PutCertificateCommand for example - where else?) can this be null? If not, then maybe let's add a validation there?

Reply:
There's also PutCertificateWithSamePinningHashCommand.
Yeah I'll add validation.
Also, I just realized this needs to be added to the cli as well.

Reply:
There's also PutLocalState which is for when we are passive.